response_operations_ui/views/accounts.py

```diff
@@ -115,19 +116,24 @@ def change_email():
                 "changed_value": form.data["email_address"],
             }
             try:
+                logger.info("Sending verification email")
```

Comment:
is there any metadata you can add to this log line?  Like a user_id or something that isn't the email in plain text?